packages/js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@livekit/protocol",
+  "version": "13.0.0",
```

Comment:
should the version match whatever the tag is? to keep things consistent?

Reply:
yeah, it will once we trigger the workflow for the first time! the `Version Package` step takes care of that.
Was just thinking it would make sense that the major version of the tag reflects our "protocol version" that we bump for changes.